src/pages/EnvironmentsPage/environmentsPermissions.ts

```diff
@@ -21,6 +22,14 @@ const makePermissionsProvider = (userEmailGetter: () => string): LeoResourcePerm
   return permissionsProvider;
 };
 
+export const makeCromwellAppsNotDeletable = (): CanDeleteProvider => {
```

Comment:
should move to the above `makePermissionsProvider` once you consolidate `CanDeleteProvider` into `LeoResourcePermissionsProvider`